js/google_ima.js

```diff
@@ -425,7 +425,8 @@ require("../html5-common/js/utils/utils.js");
           //displaying it, so just notify AMC that we are showing an overlay.
           else
           {
-            _amc.notifyNonlinearAdStarted(this.currentAMCAdPod.id);
+            // raise WILL_PLAY_NONLINEAR_AD event and notify nonlinear ad started
+            _amc.sendURLToLoadAndPlayNonLinearAd(this.currentAMCAdPod, this.currentAMCAdPod.id, this.currentAMCAdPod.ad.tag_url);
```

Comment:
If it doesn't do we need to send ad.tag_url? If not we should send null and update the comment to indicate it is alerting the AMC and player that a non linear ad is started but it is rendered by ima.